taurus/entity/dict_serializable.py

```diff
@@ -82,7 +82,40 @@ def build(d):
         return loads(dumps(d))
 
     def __repr__(self):
-        return str(self.as_dict())
+        object_dict = self.as_dict()
+        # as_dict() skips over keys in `skip`, but they should be in the representation.
+        skipped_keys = {x.lstrip('_') for x in vars(self) if x in self.skip}
+        for key in skipped_keys:
+            skipped_field = self.__getattribute__(key)
```

Comment:
Typically I think you would use the equivalent `getattr(self, key)` syntax for this.

Reply:
Done